server.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+
+app.get('/ping',(req,res)=>{
+    res.send("pong")
+})
+
+app.listen(3000,()=>{
```

Comment:
Consider using environment variables for the port number. This will make the application more flexible as the port can be easily changed without modifying the code. [important]